docker-compose.yml

```diff
@@ -15,732 +15,779 @@
+  conda-python-pandas:
+    # document to use --no-cache in case of PANDAS=master
+    image: ${ORG}/${ARCH}-conda-latest-python-${PYTHON}-pandas-${PANDAS}:latest
     build:
       context: .
-      dockerfile: cpp/Dockerfile.ubuntu-bionic
+      dockerfile: ci/docker/conda-latest-python-pandas.dockerfile
+      cache_from:
+        - ${ORG}/${ARCH}-conda-latest-python-${PYTHON}-pandas-${PANDAS}:latest
+      args:
+        org: ${ORG}
+        arch: ${ARCH}
+        conda: latest
+        python: ${PYTHON}
+        pandas: ${PANDAS}
+    shm_size: *shm-size
+    volumes: *conda-volumes
+    entrypoint: *conda-entrypoint
+    command: *python-conda-command
+
+  conda-python-dask:
+    # document to use --no-cache in case of DASK=master
+    image: ${ORG}/${ARCH}-conda-latest-python-${PYTHON}-dask-${DASK}:latest
+    build:
+      context: .
+      dockerfile: ci/docker/conda-latest-python-dask.dockerfile
+      cache_from:
+        - ${ORG}/${ARCH}-conda-latest-python-${PYTHON}-dask-${DASK}:latest
+      args:
+        org: ${ORG}
+        arch: ${ARCH}
+        conda: latest
+        python: ${PYTHON}
+        dask: ${DASK}
+    shm_size: *shm-size
+    volumes: *conda-volumes
+    entrypoint: *conda-entrypoint
+    command:
+      ["/arrow/ci/scripts/cpp_conda_build.sh /arrow /build/cpp &&
+        /arrow/ci/scripts/python_build.sh /arrow &&
+        /arrow/ci/scripts/integration_dask.sh"]
+
+  conda-python-turbodbc:
+    # document to use --no-cache in case of TURBODBC=master
+    image: ${ORG}/${ARCH}-conda-latest-python-${PYTHON}-turbodbc-${TURBODBC}:latest
+    build:
+      context: .
+      dockerfile: ci/docker/conda-latest-python-turbodbc.dockerfile
+      cache_from:
+        - ${ORG}/${ARCH}-conda-latest-python-${PYTHON}-turbodbc-${TURBODBC}:latest
+      args:
+        org: ${ORG}
+        arch: ${ARCH}
+        conda: latest
+        python: ${PYTHON}
+        turbodbc: ${TURBODBC}
+    shm_size: *shm-size
+    volumes: *conda-volumes
+    entrypoint: *conda-entrypoint
+    command:
+      ["/arrow/ci/scripts/cpp_conda_build.sh /arrow /build/cpp &&
+        /arrow/ci/scripts/python_build.sh /arrow &&
+        /arrow/ci/scripts/integration_turbodbc.sh /turbodbc /build/turbodbc"]
+
+  ########################## Python Wheels ####################################
+
+  centos-python-manylinux1:
+    image: ${ORG}/amd64-centos-python-manylinux1:latest
+    build:
+      context: python/manylinux1
+      dockerfile: Dockerfile-x86_64_base
+      cache_from:
+        - ${ORG}/amd64-centos-python-manylinux1:latest
+    shm_size: *shm-size
     environment:
-      PARQUET_TEST_DATA: /arrow/cpp/submodules/parquet-testing/data
-    volumes: *ubuntu-bionic-volumes
+      PYARROW_PARALLEL: 3
+      PYTHON_VERSION: ${PYTHON_VERSION:-3.6}
+      UNICODE_WIDTH: ${UNICODE_WIDTH:-16}
+    volumes: &manylinux-volumes
+      - .:/arrow:delegated
+      - ./python/manylinux1:/io:delegated
+    command: &manylinux-command /io/build_arrow.sh
```

Comment:
The build scripts for manylinux are like a little snowflake :(

Reply:
Added to the nightly list in the description.